client/components/themes-list/test/__snapshots__/index.jsx.snap

```diff
@@ -31,7 +31,9 @@ exports[`ThemesList should render a div with a className of "themes-list" 1`] =
       <p
         class="pattern-assembler-cta__subtitle"
       >
-        Can't find something you like? Jump right into the editor to design your homepage from scratch.
+        Can’t find something you like? 
```

Comment:
The snapshots are very sensitive to whitespace differences, so make sure they are identical!

Reply:
While we're at it -- does it matter if we use `'` or `’`? I can't find any resources / guidelines on that for Automattic.